libmozdata/config.py

```diff
@@ -72,3 +81,8 @@ def get(section, option, default=None, type=str, required=False):
     if required:
         assert value is not None, f"Option {option} in section {section} is not set"
     return value
+
+
+def set_value(section, option, value):
```

Comment:
Similar to https://github.com/mozilla/libmozdata/pull/253#discussion_r1815068094, this would be `set_default_value()`.